kwalitee/release.py

```diff
@@ -0,0 +1,40 @@
+import json, re
+import requests
+
+class Release():
+
+    def __init__(self, json):
+        self.tag = json['tag_name']
+        self.projects = self.__get_projects(json)
+
+    def __str__(self):
+        return f'<Release {self.tag}>'
+
+    def to_json(self):
+        desc = {}
+        desc['tag'] = self.tag
+        desc['projects'] = self.projects
+        return desc
+
+    def __get_projects(self, json_data):
+        markdown_desc = json_data['body']
```

Comment:
OK, so you're parsing the changelog that we post with every ocrd_all release. This works because we generate the changelog semi-automatically but I think it would be better to inspect the ocrd_all repository directly, i.e.

1. Determine the commit A of the previous release
2. Determine the commit B of the next release
3. Inspect which submodules in ocrd_all had changes committed between A and B

Reply:
Good point. My somewhat naive approach was "see what the release API offers", so I came up with this. But it surely works without API requests (and probably even more reliably).

Reply:
Since our sprint ends today, @paulpestov needs the data for his tasks in the next sprint and the current approach works, I'll take the liberty and make a new issue for implementing your suggestion.